src/main/kotlin/uk/gov/justice/digital/hmpps/communitypaybackapi/service/AppointmentOutcomeValidationService.kt

```diff
@@ -1,30 +1,49 @@
 package uk.gov.justice.digital.hmpps.communitypaybackapi.service
+import org.springframework.beans.factory.annotation.Value
 import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.communitypaybackapi.dto.AppointmentDto
 import uk.gov.justice.digital.hmpps.communitypaybackapi.dto.BadRequestException
 import uk.gov.justice.digital.hmpps.communitypaybackapi.dto.UpdateAppointmentOutcomeDto
 import uk.gov.justice.digital.hmpps.communitypaybackapi.entity.ContactOutcomeEntityRepository
 import uk.gov.justice.digital.hmpps.communitypaybackapi.service.internal.validateLengthLessThan
 import uk.gov.justice.digital.hmpps.communitypaybackapi.service.internal.validateNotNull
 import java.time.Duration
+import java.time.LocalDate
 
 @Service
 class AppointmentOutcomeValidationService(
```

Comment:
Not for this PR, but maybe we can think of some more names for classes like this one, wuch as "AppointmentOutcomeValidator" - rather than everything being a service,